cmd/helm/uninstall.go

```diff
@@ -74,7 +74,7 @@ func newUninstallCmd(c helm.Interface, out io.Writer) *cobra.Command {
 	f := cmd.Flags()
 	f.BoolVar(&o.dryRun, "dry-run", false, "simulate a uninstall")
 	f.BoolVar(&o.disableHooks, "no-hooks", false, "prevent hooks from running during uninstallation")
-	f.BoolVar(&o.purge, "purge", false, "remove the release from the store and make its name free for later use")
+	f.BoolVar(&o.purge, "purge", true, "remove the release from the store and make its name free for later use")
```

Comment:
While I agree with the idea of purging by default but, the flag name and text surrounding it don't quite make sense anymore. For the user experience I like Martins idea of making it a `--keep-history` and having docs that align with that.